manager/controlapi/ca_rotation_test.go

```diff
@@ -0,0 +1,771 @@
+package controlapi
+
+import (
+	"context"
+	"crypto/x509"
+	"io/ioutil"
+	"reflect"
+	"testing"
+	"time"
+
+	"os"
+
+	"github.com/cloudflare/cfssl/helpers"
+	"github.com/cloudflare/cfssl/initca"
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/ca"
+	"github.com/docker/swarmkit/ca/testutils"
+	"github.com/docker/swarmkit/manager/state/store"
+	digest "github.com/opencontainers/go-digest"
```

Comment:
Good point, thanks!  My auto-importer insists on adding it every time I save.  :)